docs/Modules/ecdsa/ecdsasdk.mdx

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_label: "SDK Usage"
+sidebar_position: 3
+description: "Learn to integrate the ECDSA Module using Biconomy SDK"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# ECDSA Ownership Module Integration Guide
+
+## Introduction 📘
+
+Integrate the ECDSA Validation Module into Biconomy Smart Accounts using the SDK.
+
+## Setup and Installation 🛠️
+
+<Tabs>
+<TabItem value="js" label="npm">
+
+```bash
+
+npm install @biconomy/modules ethers
+
+```
+
+</TabItem>
+<TabItem value="py" label="yarn">
+
+```bash
+
+yarn add @biconomy/modules ethers
+
+```
+
+</TabItem>
+<TabItem value="java" label="pnpm">
+
+```bash
+
+pnpm add @biconomy/modules ethers
+
+```
+
+</TabItem>
+</Tabs>
+
+## Creating a Signer 🗝️
+
+```javascript
+import { ethers } from "ethers";
+
+const provider = new ethers.providers.JsonRpcProvider("[RPC_Endpoint]");
+const signer = new ethers.Wallet("[Private_Key]", provider);
+```
+
+## Importing and Initializing ECDSA Module 🌟
+
+```javascript
+import {
+  ECDSAOwnershipValidationModule,
+  DEFAULT_ECDSA_OWNERSHIP_MODULE,
+} from "@biconomy/modules";
+
+const ecdsaModule = await ECDSAOwnershipValidationModule.create({
+  signer: signer,
+  moduleAddress: DEFAULT_ECDSA_OWNERSHIP_MODULE,
+});
+```
+
+## Creating a Smart Account 🚀
+
+```javascript
+import { BiconomySmartAccountV2 } from "@biconomy/account";
+
+const smartAccount = await BiconomySmartAccountV2.create({
+  signer: signer,
```

Comment:
Would be great to explain that signer object is based on EOA that will be owning Smart Account via ECDSA Module 
so Signer is in fact owner of the SA

Reply:
Added in the info / warning below